scripts/prepare-release.sh

```diff
@@ -68,6 +68,9 @@ sed s"!node-feature-discovery/v.*/!node-feature-discovery/$release/!" -i README.
 echo Patching '*.yaml.template' to use $container_image
 sed -E s",^([[:space:]]+)image:.+$,\1image: $container_image," -i *yaml.template
 
+# Patch Helm chart
+sed -e s"/appVersion:.*/appVersion: $release/" -i deployment/node-feature-discovery/Chart.yaml
```

Comment:
This is good. From the other comments I realized that we need to patch the `repository` and `pullPolicy` fields, too